Modules/CTP/src/RawDataQcTask.cxx

```diff
@@ -69,18 +69,18 @@ void CTPRawDataReaderTask::startOfActivity(const Activity& activity)
   mHistoClassRatios->Reset();
   mHistoInputRatios->Reset();
   mHistoMTVXBC->Reset();
-  //
+
   mRunNumber = activity.mId;
   mTimestamp = activity.mValidity.getMin();
-  //
-  // mTimestamp = 1714315086649;
-  //
+
   std::string MBclassName = mCustomParameters["MBclassName"];
   if (MBclassName.empty()) {
     MBclassName = "CMTVX-B-NOPF";
   }
   std::string run = std::to_string(mRunNumber);
-  o2::ctp::CTPRunManager::setCCDBHost("https://alice-ccdb.cern.ch");
+  std::string ccdbName = mCustomParameters["ccdbName"];
+
+  o2::ctp::CTPRunManager::setCCDBHost(ccdbName);
```

Comment:
I suggest to check if `ccdbName` is empty and use a default value (like `"https://alice-ccdb.cern.ch"`) in this case.
Like it is done for `MBclassName`

Reply:
I should probably provide a way to get it from the "official" item in the config.

Reply:
https://its.cern.ch/jira/browse/QC-1194